crates/driver/src/print_build.rs

```diff
@@ -236,36 +238,39 @@ where
         }
         if matched_paths.len() > 1 {
             return Err(anyhow::anyhow!(
-                "Found two many paths for {}, at least: {:?}",
+                "Found too many paths for {}, at least: {:?}",
                 element,
                 matched_paths
             ));
         }
         if module_config.is_none() {
-            let (path_type, path_prefix) = matched_paths.get(0).unwrap();
-            module_config = Some((v, path_type, path_prefix));
+            module_config = Some(v);
         } else {
             return Err(anyhow::anyhow!("Multiple configurations matched for {}, at least: {:?}; module config was before: {:?}", element, matched_paths, module_config));
         }
     }
-    let (module_config, path_type, _matched_prefix) = if let Some(a) = module_config {
+    let module_config = if let Some(a) = module_config {
         a
     } else {
-        return Err(anyhow!(
-            "Unable to find any matching configuration for {}",
-            element
-        ));
+        let t: TargetEntries = Default::default();
+        return Ok(t);
```

Comment:
nit: `return Ok(Default::default());` or maybe even `return Default::default();` would work?

Reply:
Addressed in 9a9ed9cecef933e111d92232c11b257c095e1cc5